package.json

```diff
@@ -24,45 +24,46 @@
     "release:publish:ci": "./scripts/release/publish.js",
     "release:version": "./scripts/release/version.js"
   },
+  "//": "Currently can't upgrade TypeScript to v4.4.2 because it breaks Rollup: https://git.io/JuCcs",
```

Comment:
`"//"` is a [safe and encouraged way](https://stackoverflow.com/a/14221781) to add comments to `package.json`.

I felt this was appropriate because otherwise it may be confusing for the next person why we're stuck on TypeScript v4.3.5.